Dockerfile.pix4d

```
@@ -48,10 +50,9 @@ RUN apt-get update && apt-get install -y software-properties-common \
 ENV PYENV_ROOT=/usr/local/.pyenv \
   PATH="/usr/local/.pyenv/bin:$PATH"
 RUN git clone https://github.com/pyenv/pyenv.git /usr/local/.pyenv \
-  && cd /usr/local/.pyenv && git checkout 9ee109b66148bc39a685926050b7b56cb4bb184b && cd - \
-  && pyenv install 3.9.2 \
-  && pyenv install 2.7.18 \
-  && pyenv global 3.9.2
+  && cd /usr/local/.pyenv && git checkout 9ee109b66148bc39a685926050b7b56cb4bb184b && cd -
+
+RUN for ver in $PYTHON_VERSIONS; do pyenv install $ver; done && pyenv global 3.9.2
```

Well, it will break when PYTHON_VERSIONS changes to replace 3.9.2 with another version so we won't miss it but you can automatically pick the last value with:
```suggestion
RUN for ver in $PYTHON_VERSIONS; do pyenv install $ver; done && pyenv global ${PYTHON_VERSION##* }
```

`${foo##* }` means "strip longest prefix of variable foo that match `* ` (any character and a space).

---

Done as suggested. Thanks.